### PR TITLE
Update clone instructions

### PR DESCRIPTION
**Pull request by gavin-stackrox**
## Description

Per title. (This is a copy of #1152) 

## Checklist
- [x] Investigated and inspected CI test results

## Testing Performed

n/a

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -104,25 +104,17 @@ The following tools are necessary to test code and build image(s):
 # Create a GOPATH: this is the location of your Go "workspace".
 # (Note that it is not – and must not – be the same as the path Go is installed to.)
 # The default is to have it in ~/go/, or ~/development, but anything you prefer goes.
-# Whatever you decide, create the directory, and add a line in your ~/.bash_profile
-# exporting the env variable:
+# Whatever you decide, create the directory, and set GOPATH and update PATH:
```
Comment (vikin91) [inline README.md:107]:
```suggestion
# Whatever you decide, create the directory, set GOPATH, and update PATH:
```

File: README.md [hunk 2]
```diff
@@ -104,25 +104,17 @@ The following tools are necessary to test code and build image(s):
...
 export GOPATH=$HOME/go # Change this if you choose to use a different workspace.
 export PATH=$PATH:$GOPATH/bin
-source ~/.bash_profile
+# You will probably want to permanently set these by adding those commands to your shell
```
Comment (vikin91) [inline README.md:110]:
```suggestion
You probably want to permanently set these by adding the following commands to your shell
```

**Review by vikin91 — APPROVED** [review]
LGTM! Just found some minor text issues

File: README.md [hunk 3]
```diff
@@ -104,25 +104,17 @@ The following tools are necessary to test code and build image(s):
...
+# configuration (e.g. ~/.bash_profile)
 
-$ cd $GOPATH
-$ mkdir bin pkg src
-
-# Replace https git-urls with ssh, required to fetch go dependencies.
-$ git config --global --add url.git@github.com:.insteadof https://github.com/
-
-# Instruct Go to bypass the Go package proxy for our private dependencies
-$ go env -w GOPRIVATE=github.com/stackrox
```
[the remark this answers is not in the document]
Comment (gavin-stackrox) [reply README.md:120]: > Would the thing still work without these lines now?

It did for me in a clean Ubuntu env.

File: README.md [hunk 4]
```diff
@@ -104,25 +104,17 @@ The following tools are necessary to test code and build image(s):
 # Create a GOPATH: this is the location of your Go "workspace".
 # (Note that it is not – and must not – be the same as the path Go is installed to.)
 # The default is to have it in ~/go/, or ~/development, but anything you prefer goes.
-# Whatever you decide, create the directory, and add a line in your ~/.bash_profile
-# exporting the env variable:
+# Whatever you decide, create the directory, set GOPATH, and update PATH:
 export GOPATH=$HOME/go # Change this if you choose to use a different workspace.
 export PATH=$PATH:$GOPATH/bin
-source ~/.bash_profile
+You probably want to permanently set these by adding the following commands to your shell
```
Comment (msugakov) [inline README.md:110]:
```suggestion
# You probably want to permanently set these by adding the following commands to your shell
```

File: README.md [hunk 5]
```diff
@@ -104,25 +104,17 @@ The following tools are necessary to test code and build image(s):
...
-
-# Instruct Go to bypass the Go package proxy for our private dependencies
-$ go env -w GOPRIVATE=github.com/stackrox
-
-$ cd $GOPATH
-$ mkdir -p src/github.com/stackrox
-$ cd src/github.com/stackrox
-$ git clone git@github.com:stackrox/stackrox.git
+cd $GOPATH
+mkdir -p bin pkg src
```
Comment (msugakov) [inline README.md:114]:
```suggestion
mkdir -p bin pkg
```
Since `src` is also mentioned below.

**Review by msugakov — APPROVED** [review]
No blockers from my side.

**Pull request merged**